app/models/geocode.rb

```diff
@@ -0,0 +1,16 @@
+module Geocode
```

Comment:
I'd name this `Geocodable`so it sounds as though you're extending the including classes with that _behaviour_